File: SortColors_Brute.cpp
#include <iostream>
#include <vector>
using namespace std;

void sortColors(vector<int>& nums) {
    int count0 = 0, count1 = 0, count2 = 0;

    for (int num : nums) {
        if (num == 0) count0++;
        else if (num == 1) count1++;
        else count2++;
    }

    int i = 0;
    while (count0--) nums[i++] = 0;
    while (count1--) nums[i++] = 1;
    while (count2--) nums[i++] = 2;
}

void print(vector<int> arr) {
    for (int num : arr) cout << num << " ";
    cout << endl;
}

int main() {
    vector<vector<int>> tests = {
        {2,0,2,1,1,0},
        {2,0,1},
        {0},
        {1,2,0}
    };

    for (auto& test : tests) {
        sortColors(test);
        print(test);
    }
}

File: SortColors_Brute.java
public class MaxSubarray_Brute {
    public static int maxSubArray(int[] nums) {
        int maxSum = Integer.MIN_VALUE;
        for (int i = 0; i < nums.length; i++) {
            int sum = 0;
            for (int j = i; j < nums.length; j++) {
                sum += nums[j];
                maxSum = Math.max(maxSum, sum);
            }
        }
        return maxSum;
    }

    public static void main(String[] args) {
        int[][] tests = {
            {-2,1,-3,4,-1,2,1,-5,4},
            {1},
            {5,4,-1,7,8},
            {-1,-2,-3,-4}
        };

        for (int[] test : tests) {
            System.out.println("Output: " + maxSubArray(test));
        }
    }
}
